### PR TITLE
Week 4 Synthesizing Tasks

### PR DESCRIPTION
**Pull request by cheongwoo**
I am having hard time putting together REPORT.  I can't seem to add line break without causing an error.

### CODE REVIEW

**Comment by cheuschober** [comment]
@cheongwoo, line breaks are just `'\n'`, eg, `'first line\nsecond line'`

**Comment by cheongwoo** [comment]
Even when I put \n for line break I get error message saying it's unexpected character.  Should entire report be saved to REPORT variable?  I just can't figure out how to save 7 line output into one variable.  I can't even find example in google.  Anyway I type it I get error it seems.  At the end of first line of code I get error saying it's unexpected character.  Is there any proper way to end a line of code?  I thought \ was the right way to end one line before continuing to next.  But I still get error.

Thank you!

File: task_03.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        if PRE_QUALIFIED == 'y':
+            RATE = decimal.Decimal('2.05')
+    elif DURATION <= 20 and DURATION >= 16:
+        if PRE_QUALIFIED == 'y':
+            RATE = decimal.Decimal('2.62')
+
+TOTAL = PRINCIPAL * (1 + fractions.Fraction(RATE, 100) / 12) ** (12 * DURATION)
+TOTAL = round(TOTAL)
+
+REPORT = 'Loan Report for: ' + NAME\n\
```
Comment (cheuschober) [inline task_03.py]: @cheongwoo, You're missing operators between the lines! Plus, `\n` must be inside a string. Let's break this down a bit more simply.

    MULTI = ('Hello {1}. I see you need to break right here\nacross multiple lines. It makes me {2}.\nTry '
             '{3} and your lines can continue to {4}')
    MULTI = MULTI.format(1, 2, 3, 'a', 'b' ...)

File: task_01.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    HIGHLIGHT1 = 'Off-White'
+    HIGHLIGHT2 = 'Paper White'
+elif ACCENT == 'Platinum Mist':
+    HIGHLIGHT1 = 'Bone White'
+    HIGHLIGHT2 = 'Just White'
+elif ACCENT == 'Spartan Sage':
+    HIGHLIGHT1 = 'Fractal White'
+    HIGHLIGHT2 = 'Not White'
+
+HIGHLIGHT = raw_input('Which highlight color, "{H1}" or "{H2}"?: '
```
Comment (cheuschober) [inline task_01.py]: @cheongwoo, A very interesting approach to this problem -- in some ways more elegant than the reference example I showed in the videos! Great job!

File: task_03.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+DURATION = raw_input('For how many years is this loan being borrowed? ')
+DURATION = int(DURATION)
+PRE_QUALIFIED = raw_input('Are you pre-qualified for this loan? ')
+PRE_QUALIFIED = PRE_QUALIFIED.lower()
+PRE_QUALIFIED = PRE_QUALIFIED[:1]
+
+if PRINCIPAL <= 199999 and PRINCIPAL >= 0:
+    if DURATION <= 15 and DURATION >= 1:
+        if PRE_QUALIFIED == 'y':
+            RATE = decimal.Decimal('3.63')
```
Comment (cheuschober) [inline task_03.py]: To save yourself the trouble of all the decimal.Decimal() conversions you could have saved that till after your big chain of IF's

File: task_03.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    TOTAL = None
+
+if PRE_QUALIFIED == 'y':
+    PRE_QUALIFIED = 'Yes'
+elif PRE_QUALIFIED == 'n':
+    PRE_QUALIFIED = 'No'
+
+REPORT = ('Loan Report for: ' + NAME +
+          '\n-------------------------------------------------------------' +
+          '\n      {:<19}'.format('Principal:') + '$' + '{:>7,}'.
```
Comment (cheuschober) [inline task_03.py]: Nice report!

**Pull request merged**